specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/examples/Diagnostics_ListHostingEnvironmentDetectorResponses.json

```diff
@@ -8,22 +8,17 @@
     "responses": {
         "200": {
             "body": {
-                "value": [
-                    {
-                        "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/Sample-WestUSResourceGroup/providers/Microsoft.Web/hostingEnvironments/SampleAppServiceEnvironment/detectors/runtimeavailability",
-                        "name": "runtimeavailability",
-                        "properties": {
-                            "metadata": {
-                                "description": "This detector analyzes all the requests to all applications running on this app service environment.",
-                                "category": "Availability and Performance",
-                                "subCategory": "App Availability",
-                                "supportTopicId": ""
-                            },
-                            "dataset": [
-                            ]
-                        }
-                    }
-                ]
+                "id": "/subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/Sample-WestUSResourceGroup/providers/Microsoft.Web/hostingEnvironments/SampleAppServiceEnvironment/detectors/runtimeavailability",
```

Comment:
the "value" here is actually correct since this is from a list operation.

Reply:
Removed it from the wrong file, added it back to this file and removed it from Diagnostics_GetHostingEnvironmentDetectorResponse.